drivers/overlay/overlay.go

```diff
@@ -1750,11 +1750,17 @@ func (d *Driver) get(id string, disableShifting bool, options graphdriver.MountO
 		absLowers = newAbsDir
 	}
 
+	lowerDirs := strings.Join(absLowers, ":")
+	if len(composeFsLayers) > 0 {
+		composeFsLayersLowerDirs := strings.Join(composeFsLayers, "::")
+		lowerDirs = lowerDirs + "::" + composeFsLayersLowerDirs
+	}
+
```

Comment:
If I'm reading this right, `lowerDirs` and `absLowers` no longer refer to the same set of locations, which is a problem later if we find ourselves using `absLowers`.

Reply:
yes after this point absLowers cannot be used anymore as it misses the data only directories.

Do you suggest setting it to `nil` so we will catch in future any error if we attempt to use it or any other suggestions?

Reply:
I think it would make sense to nil it out.

Reply:
done, I am setting it to `nil` after usage